packages/core/src/extractors/prompts.ts

```diff
@@ -0,0 +1,81 @@
+export interface DefaultPromptTemplate {
+  contextStr: string;
+}
+
+export interface DefaultKeywordExtractorPromptTemplate
+  extends DefaultPromptTemplate {
+  keywords: number;
+}
+
+export interface DefaultQuestionAnswerPromptTemplate
+  extends DefaultPromptTemplate {
+  numQuestions: number;
+}
+
+export interface DefaultNodeTextTemplate {
+  metadataStr: string;
+  content: string;
+}
+
+export const defaultKeywordExtractorPromptTemplate = ({
+  contextStr = "",
+  keywords = 5,
+}: DefaultKeywordExtractorPromptTemplate) => `
+  ${contextStr}. Give ${keywords} unique keywords for thiss
+  document. Format as comma separated. Keywords:
+`;
+
+export const defaultTitleExtractorPromptTemplate = (
+  { contextStr = "" }: DefaultPromptTemplate = {
+    contextStr: "",
+  },
+) => `
+  ${contextStr}. Give a title that summarizes all of the unique entities, titles or themes found in the context. Title:
+`;
+
+export const defaultTitleCombinePromptTemplate = (
+  { contextStr = "" }: DefaultPromptTemplate = {
+    contextStr: "",
+  },
+) => `
+  ${contextStr}. Based on the above candidate titles and content,s
```

Comment:
@EmanuelCampos these prompts still need to be fixed. Can you open a new PR to do so?